integration-tests/test-suite/partials/layout.xml

```diff
@@ -532,5 +532,62 @@
 		</urls>
 	</test>
 
+	<test name="S-comparator-Layout-Auto-Height-1px-Page">
+		<collect>
+			<open/>
+			<resolution width="767"/>
+			<screen/>
+		</collect>
+		<compare>
+			<screen comparator="layout"/>
+		</compare>
+		<urls>
+			<url href="comparators/layout/1px_height_page.jsp"/>
+		</urls>
+	</test>
+
+	<test name="S-comparator-Layout-Auto-Height-0px-Page">
```

Comment:
I am not 100% sure but with change in integration tests you probably should update also https://github.com/Cognifide/aet/blob/master/integration-tests/sanity-functional/src/test/resources/features/filtering.feature or/and https://github.com/Cognifide/aet/blob/master/integration-tests/sanity-functional/src/test/java/com/cognifide/aet/sanity/functional/HomePageTilesTest.java

I remember that documentation contains some tutorial how to run these tests but unfortunatelly I cannot find it :P